app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Service catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization, both {{site.base_gateway}} services and external integration resources. This catalog is the single source of truth for your organization’s service inventory and their dependencies.
+
+Specifically, Service Catalog addresses the following problems:
```

Comment:
The language in this section is too close to the marketing communication. Some of these, in the way they are written aren't really 'problems', so for example this sentence positions 'service ownership' as more of a feature: 

Service ownership: Keep track of your organization’s service ownership by mapping teams and services.

The problem here would be something around organizations having to track hundreds of services across hundreds of teams. 

Maybe a use case table would be a better fit?